syntax = "proto3";

/**
 * This is an example of a protobuf message.
 */
message SessionKeyRequest {
    bytes session_hash = 1;
}

message SessionKeyExpire {
    bytes session_hash = 1;
}

message SessionKeyResponse {
    bytes session_hash = 1;
    bytes session_key = 2;
}


